amti/actions/review.py

```diff
@@ -113,6 +138,9 @@ def review_batch(
         the path to the directory for the batch.
     approve_all : bool
         a flag to decide approve all submissions
+    mark_file : str
+        the name of marked assignments file saved as a json
+        if None outputs to STDOUT
```

Comment:
Renaming the argument (comment at amti/actions/review.py#L129-L130) and removing the logic to construct the path manually (comment at amti/actions/review.py#L157-L159) will require an update to the doc string. Probably something like:

```
mark_file_path : str
    The path at which to save the assignment marks.
```

Reply:
Done